.github/workflows/maven-release.yml

```diff
@@ -31,15 +31,24 @@ on:
         type: string
         required: true
 
+      ref:
+        description: |
+          The git reference (branch, tag, or SHA) to check out when publishing the release.
+          Defaults to 'main'.
+        required: false
+        type: string
+        default: 'main'
```

Comment:
Do both `"..."` and `'...'` work here? Asking because, I see staging in double quotes and main in single quotes.